src/LibraryManager.Vsix/UI/Converters/CheckBoxAutomationNameConverter.cs

```diff
@@ -9,20 +9,25 @@ internal class CheckBoxAutomationNameConverter : IMultiValueConverter
     {
         public object Convert(object[] values, Type targetType, object parameter, CultureInfo culture)
         {
-            if (values == null || values.Length != 2 || !(values[0] is string) || !(values[1] is bool))
+            if (values == null || values.Length != 3 || !(values[0] is string))
             {
                 return null;
```

Comment:
Converters should return `DependencyObject.UnsetValue` when the inputs are invalid in an anticipated way, or throw otherwise (see [here](https://docs.microsoft.com/en-us/dotnet/api/system.windows.data.imultivalueconverter.convert?view=netframework-4.8#remarks)).  Returning null doesn't signal that there was anything wrong with the converter inputs, so it would be harder to track down that's something's going wrong.

Reply:
I kinda lean towards throwing if the arguments are not in the right order or missing, but you could go either way with it.  It's handy to see the exceptions being thrown when running under a debugger if a control doesn't behave as expected.

Reply:
I like throwing exceptions better. Made changes here:
https://github.com/aspnet/LibraryManager/commit/0d5e1914b17d0fe7853d0b50b666ee02aede15c4